themes/startwords/theme.toml

```diff
@@ -1,12 +1,12 @@
-name = "Blank"
+name = "startwords"
 license = "MIT"
-licenselink = "https://github.com/vimux/blank/blob/master/LICENSE"
-description = "Starter Hugo theme for developers."
+licenselink = "https://github.com/Princeton-CDH/startwords/blob/master/themes/startwords/LICENSE"
+description = "Theme for the Startwords research periodical."
 homepage = "https://github.com/vimux/blank/"
```

Comment:
Is this the theme homepage? Should it point to the theme within the startwords repo?

Reply:
oh, good catch! not sure what it should really do since I guess our theme isn't published on GitHub on its own. I guess we could also just remove it?

Reply:
yeah, I like that idea even better